client/src/main/java/tds/assessment/Assessment.java

```diff
@@ -85,4 +87,54 @@ public float getStartAbility() {
     public String getSubjectName() {
```

Comment:
The reason I used subjectName was because there are other columns in that table like key, client, and loadconfig.  That being said we can just change the code later if we need more than name.